jobpy/glassdoor.py

```diff
@@ -1,148 +1,149 @@
 ## scrapes data off glassdoor.ca and pickles it
 
-import pickle
 import logging
 import requests
 import bs4
 import re
 import os
-import sys
 from math import ceil
 
+from .jobpy import JobPy
 from .config.settings import MASTERLIST_HEADER
 from .tools.tools import filter_non_printables
 from .tools.tools import post_date_from_relative_post_age
 
-sys.path.append('../')
-
-# the maximum monster search results per page
-glassdoor_max_results_per_page = 30
-
-
-def convert_glassdoor_radius(radius):
-    """function that converts the user input radius to an available glassdoor radius"""
-    # glassdoor only accepts discrete radius values of 10, 20, 30, 50 and 100 kilometers
-    if radius < 10:
-        radius = 0
-    elif radius >= 10 and radius < 20:
-        radius = 10
-    elif radius >= 20 and radius < 30:
-        radius = 20
-    elif radius >= 30 and radius < 50:
-        radius = 30
-    elif radius >= 50 and radius < 100:
-        radius = 50
-    elif radius >= 100:
-        radius = 100
-
-    glassdoor_radius = {
-        0: 0,
-        10: 6,
-        20: 12,
-        30: 19,
-        50: 31,
-        100: 62
-    }
-
-    return glassdoor_radius[radius]
-
-
-def scrape_glassdoor_to_pickle(jobpy_obj):
-    """function that scrapes job posting from glassdoor and pickles it"""
-    try:
+
+class GlassDoor(JobPy):
+
+    def __init__(self, args):
+        super().__init__(args)
```

Comment:
should it be `super(JobPy, self).__init__(args)`?

Reply:
I believe this is the python3 way